packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -251,14 +251,16 @@ export async function createHotReloaderTurbopack(
       return () => {}
     }
     if (buildingIds.size === 0) {
-      consoleStore.setState(
-        {
-          loading: true,
-          trigger: id,
-          url: requestUrl,
-        } as OutputState,
-        true
-      )
+      if (opts.nextConfig.logging !== false) {
```

Comment:
you can set `logging: !!nextConfig.logging` into consoleStore to indicate if you enable logging or not.